src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,93 @@
+package duke.command;
+
+import duke.ui.Ui;
+import duke.tasklist.TaskList;
+import duke.storage.Storage;
+import duke.task.Deadlines;
+import duke.task.Tasks;
+import duke.task.Todos;
+import duke.task.Events;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+/**
+ * Represents a command generalised for the addition of tasks. A <code>AddCommand</code> object corresponds to
+ * a command represented by the input command and the arguments by the user e.g., <code>todos, Eat a cake.</code>
+ */
+public class AddCommand extends Commands {
+    public static final String COMMAND_TODO = "todo";
+    public static final String COMMAND_EVENT = "event";
+    public static final String COMMAND_DEADLINE = "deadline";
+    public static final String SUCCESS_MESSAGE = "    Command Executed Successfully";
+    public static final String FAILURE_MESSAGE
+            = "    'Add' Command Executed Unsuccessfully";
+
+    private static final boolean IS_EXIT = false;
+    private final String commandWord;
+    private final String arguments; // In the form of user duke.command
```

Comment:
Not sure whether an inline comment violates the coding standard or not. Perhaps you can put the comments above the attribute declaration?